### PR TITLE
TiDB monitoring: add table storage info status api

### PR DESCRIPTION
**Pull request by en-jin19**
### What is changed, added or deleted? (Required)
added table storage info status API in the "TiDB monitoring API" document.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: [#6706](https://github.com/pingcap/docs-cn/pull/6706)
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-monitoring-api.md [hunk 1]
```diff
@@ -8,7 +8,8 @@ aliases: ['/docs/dev/tidb-monitoring-api/']
 
 You can use the following two types of interfaces to monitor the TiDB cluster state:
 
-- [The state interface](#use-the-state-interface): this interface uses the HTTP interface to get the component information.
+- [The state interface](#running-status): this interface uses the HTTP interface to get the component information.
+- [Storage information](#storage-information): it uses the HTTP interface to get the storage information from data tables.
```
Comment (TomShawn) [inline tidb-monitoring-api.md:12]:
```suggestion
- [Storage information](#storage-information): this interface uses the HTTP interface to get the storage information of data tables.
```

File: tidb-monitoring-api.md [hunk 2]
```diff
@@ -31,6 +34,48 @@ curl http://127.0.0.1:10080/status
 }
 ```
 
+#### Storage information
+
+The following example uses `http://${host}:${port}/schema_storage/${db}/${table}` to get the storage information from the specific data tables. The result is returned in **JSON** format.
```
Comment (TomShawn) [inline tidb-monitoring-api.md:39]:
```suggestion
The following example uses `http://${host}:${port}/schema_storage/${db}/${table}` to get the storage information of the specific data table. The result is returned in **JSON** format.
```

**Review by TomShawn — APPROVED** [review]
Rest LGTM

**Review by xhebox — APPROVED** [review]

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**